src/components/course/data/constants.js

```diff
@@ -45,8 +45,8 @@ export const ENROLLMENT_COURSE_RUN_KEY_QUERY_PARAM = 'course_run_key';
 
 export const LICENSE_REQUESTED_ALERT_DISMISSED_COOKIE_NAME = 'license-requested-alert-dismissed';
 export const LICENSE_REQUESTED_ALERT_HEADING = 'Course requested';
-export const LICENSE_REQUESTED_ALERT_TEXT = 'Your organization’s subscription covers all of the courses in this catalog.'
-                                    + ' You have already requested access to all courses.';
+/* eslint-disable-next-line max-len */
+export const LICENSE_REQUESTED_ALERT_TEXT = 'Your organization’s subscription covers all of the courses in this catalog. You have already requested access to all courses.';
```

Comment:
nit: the quote in "organization’s" is unicode
```
>>> ch = "organization’s"[-2]
>>> ch.encode()
b'\xe2\x80\x99'
>>> ord(ch)
8217
```
https://www.codetable.net/decimal/8217